server/models/like.js

```diff
@@ -0,0 +1,20 @@
+const mongoose = require('mongoose');
+
+const LikeSchema = new mongoose.Schema({
+    created   : {
+		type    : Date,
+		default : Date.now
+	},
+    userID: {
```

Comment:
We reference the user by email. Because it is indexed and to be consistent with the rest of the app you can just call this property user